audiophiler/templates/base.html

```diff
@@ -44,6 +44,33 @@
                             My Files
                         </a>
                     </li>
+                    {% if file.author == auth_dict["uid"] or is_rtp or is_eboard %}
```

Comment:
Why does this check file.author? Isn't this the navbar option for selected harolds?